graviti/dataframe/frame.py

```diff
@@ -584,29 +584,29 @@ def extend(self, values: Union[Iterable[Dict[str, Any]], "DataFrame"]) -> None:
             TypeError: When the given Dataframe mismatched with the self schema.
 
         Examples:
-        >>> df = DataFrame([
-        ...     {"filename": "a.jpg", "box2ds": {"x": 1, "y": 1}},
-        ...     {"filename": "b.jpg", "box2ds": {"x": 2, "y": 2}},
-        ... ])
+            >>> df = DataFrame([
```

Comment:
commit message is not clear, shall we mention "correct the indentation in example"?